### PR TITLE
Alba healthcheck work

### PR DESCRIPTION
**Pull request by JeffreyDevloo**
Wait for namespace to be fully created
Disk safety now lists namespaces instead of safety on success

### CODE REVIEW

File: ovs/extensions/healthcheck/alba.py [hunk 1]
```diff
@@ -195,16 +193,30 @@ def check_if_proxies_work(result_handler):
...
-                        except AlbaException:
-                            raise
+                        # Create namespace
+                        AlbaCLI.run(command='proxy-create-namespace',
+                                    named_params={'host': ip, 'port': service.ports[0]},
+                                    extra_params=[namespace_key, preset_name])
+                        # Wai until fully created
+                        namespace_start_time = time.time()
+                        while True:
+                            if time.time() - namespace_start_time > 30:
```
Comment (pploegaert) [inline ovs/extensions/healthcheck/alba.py]: I'd put 30 in a constant above ...

File: ovs/extensions/healthcheck/alba.py [hunk 2]
```diff
@@ -195,16 +193,30 @@ def check_if_proxies_work(result_handler):
...
+                                raise RuntimeError('Creation namespace has timed out after {0}s'.format(time.time() - namespace_start_time))
```
Comment (pploegaert) [inline ovs/extensions/healthcheck/alba.py]: put this equation in a variable: time.time() - namespace_start_time ... = more readable

File: ovs/extensions/healthcheck/alba.py [hunk 3]
```diff
@@ -273,7 +278,16 @@ def check_if_proxies_work(result_handler):
...
-                            AlbaCLI.run(command='proxy-delete-namespace', named_params={'host': ip, 'port': service.ports[0]}, extra_params=[namespace_key])
+                            AlbaCLI.run(command='proxy-delete-namespace',
+                                        named_params={'host': ip, 'port': service.ports[0]},
+                                        extra_params=[namespace_key])
+                            namespace_delete_start = time.time()
+                            while True:
+                                namespaces = AlbaCLI.run(command='list-namespaces', config=abm_config)
+                                if namespace_key not in namespaces:
+                                    break
+                                if time.time() - namespace_delete_start > 30:
```
Comment (pploegaert) [inline ovs/extensions/healthcheck/alba.py]: add to (new) constant above

File: ovs/extensions/healthcheck/alba.py (hunk 2 above)
Comment (JeffreyDevloo) [inline ovs/extensions/healthcheck/alba.py]: Doing it in the if is actually better as it removes the small downtime of allocation the variable and calling for it.

**Pull request merged**